podcasts/Onboarding/Encourage Account Creation/Banner/InformationalBannerViewCoordinator.swift

```diff
@@ -0,0 +1,66 @@
+import PocketCastsUtils
+import PocketCastsServer
+
+class InformationalBannerViewCoordinator {
+    private var viewModel: InformationalBannerPresenting
+
+    var onDismissBanner: (() -> Void)? = nil
+
+    init(viewModel: InformationalBannerPresenting) {
+        self.viewModel = viewModel
+        setupBinding()
+    }
+
+    private func setupBinding() {
+        viewModel.onCloseBannerTap = { [weak self] in
+            self?.dismissBanner()
+        }
+
+        viewModel.onCreateFreeAccountTap = { [weak self] in
+            self?.presentLoginFlow()
+        }
+    }
+
+    func shouldShowBanner() -> Bool {
+        guard
+            FeatureFlag.encourageAccountCreation.enabled,
+            Settings.shouldShowBanner(for: viewModel.bannerType),
+            !SyncManager.isUserLoggedIn()
+        else {
+            return false
+        }
+        return true
+    }
+
+    func dismissBanner() {
+        onDismissBanner?()
+        Settings.dismissBanner(for: viewModel.bannerType)
+        Analytics.track(.informationalBannerViewDismissed, properties: ["source": viewModel.bannerType.rawValue.lowerSnakeCased()])
+    }
+
+    func presentLoginFlow() {
+        NavigationManager.sharedManager.navigateTo(NavigationManager.onboardingFlow,
+                                                   data: ["flow": OnboardingFlow.Flow.loggedOut])
+        Analytics.track(.informationalBannerViewCreateAccountTap, properties: ["source": viewModel.bannerType.rawValue.lowerSnakeCased()])
+    }
+
+    func tableHeaderView(size: CGSize, onDismissBanner: @escaping () -> Void) -> UIView? {
+        guard let viewModel = viewModel as? InformationalBannerViewModel else {
+            return nil
+        }
+        self.onDismissBanner = onDismissBanner
+        let headerView = UIView(frame: CGRect(origin: .zero, size: size))
+        let bannerView = InformationalBannerView(viewModel: viewModel).themedUIView
+        bannerView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
```suggestion
```

`anchorToAllSidesOf(view:)` [sets](https://github.com/Automattic/pocket-casts-ios/blob/fd6a3286a1a95c878fb339324573e56795e33100/Modules/Utils/Sources/PocketCastsUtils/Extensions/UI/UIViewExtension.swift#L8) `translatesAutoresizingMaskIntoConstraints = false` so I believe this can be removed if you'd like.